snap/snapcraft.yaml

```diff
@@ -27,27 +28,29 @@ apps:
       - avahi-observe
 
 parts:
+  npm:
+    source: .
+    plugin: npm
```

Comment:
how about dropping this part altogether and instead add a `stage-snaps:` entry for `node/16/stable` to the `dashkiosk`part ?

Reply:
Hmm that dumped core.

Reply:
Ah! Of course. That's because `node/16` is `base: core20`. I've pushed an update to drop this down to `base: core20`. Looking now into why I'm missing the "unassigned" group.

Reply:
> Looking now into why I'm missing the "unassigned" group.

Ah, that gets created dynamically. In that case, this is good to go.